pax-std/pax-std-primitives/src/ellipse.rs

```diff
@@ -91,23 +91,8 @@ impl InstanceNode for EllipseInstance {
 
     fn update(
```

Comment:
we can remove update entirely since there is a no-op default implementation instance node